src/ensembl/production/metadata/updater/base.py

```diff
@@ -54,3 +57,24 @@ def get_or_new_source(self, meta_session, db_type):
             )
             meta_session.add(dataset_source)  # Only add a new DatasetSource to the session if it doesn't exist
         return dataset_source
+
+    def update_attributes(self, dataset, attributes, session):
+        genebuild_dataset_attributes = []
+        for attribute, value in attributes.items():
+            meta_attribute = session.query(Attribute).filter(Attribute.name == attribute).one_or_none()
+            if meta_attribute is None:
+                # TODO: This will be removed after the 2000 species are loaded.
+                meta_attribute = Attribute(
+                    name=attribute,
+                    label=attribute,
+                    description=attribute,
+                    type="string",
+                )
```

Comment:
Is it time to remove attribute accumulation and have it dedicated on the Django now that the 2000 have been loaded? We shouldn't have much more.

Reply:
Make sense! thanks I'll remove the code and method call then.